device/globals/dialog_instance.gd

```diff
@@ -145,7 +145,7 @@ func init(p_params, p_context, p_intro, p_outro):
 
 	var parsed_ok = label.parse_bbcode(text)
 	assert(parsed_ok == OK)
-	label.bbcode_text = text
+	label.bbcode_text = "\n" + text
```

Comment:
What is this for?

Reply:
I just noticed the commit message. Godot has margins though, but position, margin and offsets have been a huge mess in the demo dialogs. In #180, which is pending testing, I fixed those issues for dd_dialogs. I haven't touched this one yet though.

Reply:
I haven't had the opportunity to look at #180 because everything in Lunar Adventure's codebase is such a mess, but if vertical alignment ("always center this, ok?") was so easy, I wonder why it isn't better documented. At least Godot doesn't have anything simple for it.

Reply:
Do you think you will achieve proper vertical alignment in #180 ? I have talked about this with @mhilbrunner, that it would be very nice to have better support for dealing with text, and it does appear to be done best in Godot itself, with that newline hack as a temporary workaround.

If this PR is otherwise fine, we can just drop that one commit and I can keep it around my branch for personal use.